docs/framework/wcf/feature-details/debugging-windows-authentication-errors.md

```diff
@@ -133,7 +133,7 @@ When using Windows authentication as a security mechanism, the Security Support
  [!code-vb[C_DebuggingWindowsAuth#3](../../../../samples/snippets/visualbasic/VS_Snippets_CFX/c_debuggingwindowsauth/vb/source.vb#3)]  
   
 #### SSPI Is Not Available  
- The following operating systems do not support Windows authentication when used as a server: [!INCLUDE[wxp](../../../../includes/wxp-md.md)] Home Edition, [!INCLUDE[wxp](../../../../includes/wxp-md.md)] Media Center Edition, and Windows Vista Home editions.  
+ Windows Vista Home editions do not support Windows authentication when used as a server.
```

Comment:
We might have to revisit this even further. Vista would also be part of the issue referenced.